packages/ag-charts-community/src/chart/series/cartesian/lineSeries.ts

```diff
@@ -543,9 +590,9 @@ export class LineSeries extends CartesianSeries<
         const { spans } = contextData.strokeData;
         const [lineNode] = paths;
 
-        lineNode.path.clear(true);
+        lineNode.path.clear();
```

Comment:
Does removal of change detection make any difference?

Reply:
I did some testing and didn't see much difference (if any) - we may have optimised out the problem this was solving elsewhere.